packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -69,7 +68,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
       {!!showTopic && <ChatHeader {...headerProps} />}
       <Stack className={chatArea} tokens={participantListContainerPadding} horizontal grow>
         <Stack className={chatWrapper} grow>
-          {props.showErrorBar ? <ErrorBar {...errorBarProps} /> : <></>}
```

Comment:
Can we make it true by default but still allow user not to show it(so keep it in options, but true by default)

Reply:
Contoso might have their own error handling outside of our our composite, I believe there should be cases they don't wanna show it, but totally agree that we show it if no one changes the options

Reply:
Seems a little extreme to force an Error Bar